Extension/src/LanguageServer/client.ts

```diff
@@ -1294,20 +1299,34 @@ export class DefaultClient implements Client {
         this.rootFolder = workspaceFolder;
         this.rootRealPath = this.RootPath ? (fs.existsSync(this.RootPath) ? fs.realpathSync(this.RootPath) : this.RootPath) : "";
 
-        let storagePath: string | undefined;
+        const baseStoragePath: string | undefined = util.getBaseStoragePath();
+        let legacyStoragePath: string | undefined;
+        let storagePath: string | undefined = "";
```

Comment:
It looks like `storagePath` can just be a string and doesn't need to be undefined too.